Description: Install released _version.py from PyPI
 The Debian package uses the GitHub sources, as these include package tests.
 This patch installs the PyPI version of _version.py (also generated by
 versioneer.py, and converted to UNIX line endings) in place of the
 git-aware _version.py in the GitHub version.
Forwarded: not-needed
Author: Julian Gilbey <jdg@debian.org>
Last-Update: 2023-06-22

--- /dev/null
+++ b/src/debugpy/_version.py
@@ -0,0 +1,21 @@
+
+# This file was generated by 'versioneer.py' (0.23) from
+# revision-control system data, or from the parent directory name of an
+# unpacked source archive. Distribution tarballs contain a pre-generated copy
+# of this file.
+
+import json
+
+version_json = '''
+{
+ "date": "2023-04-03T17:37:14-0700",
+ "dirty": false,
+ "error": null,
+ "full-revisionid": "2f3e0bb7d5984486def3f2e3d246b974cf243b50",
+ "version": "1.6.7"
+}
+'''  # END VERSION_JSON
+
+
+def get_versions():
+    return json.loads(version_json)
